com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/Breakpoint.java

```diff
@@ -325,7 +325,8 @@ private Location findMethodLocaiton(ReferenceType refType, String methodName, St
         for (Method method : methods) {
             if (!method.isAbstract() && !method.isNative()
                     && methodName.equals(method.name())
-                    && (methodSiguature.equals(method.genericSignature()) || methodSiguature.equals(method.signature()))) {
+                    && (methodSiguature.equals(method.genericSignature()) || methodSiguature.equals(method.signature())
+                            || toNoneGeneric(methodSiguature).equals(method.signature()))) {
```

Comment:
This is a good catch, it can fix https://github.com/microsoft/vscode-java-debug/issues/1414 as well.

`toNoneGeneric()` function is duplicated with `org.eclipse.jdt.core.Signature.getTypeErasure()`, we can reuse the one from the JDT core directly.